examples/fuzzed_data_provider_fuzz_test.cc

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// A fuzz target that demonstrates the use of FuzzedDataProvider.
+
+#include <fuzzer/FuzzedDataProvider.h>
+#include <cstdint>
+#include <cstddef>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+    FuzzedDataProvider fuzzed_data(Data, Size);
+
+    // Intentionally using uint16_t here to avoid empty |second_part|.
```

Comment:
I'm not sure I understand this comment. How does using `uint16_t` here prevent an empty second_part?

Reply:
Errr, I copied those codes from the official  [example](https://source.chromium.org/chromium/chromium/src/+/master:net/cert/internal/verify_name_match_fuzzer.cc;drc=0be62a8d95f7fa1455fce1a76f0fa5b8484d0c8c?originalUrl=https:%2F%2Fcs.chromium.org%2Fchromium%2Fsrc%2Fnet%2Fcert%2Finternal%2Fverify_name_match_fuzzer.cc), since the context has changed, I hardly can tell the meaning of this comment. Let me delete this line!